.github/workflows/build.yml

```diff
@@ -98,10 +98,37 @@ jobs:
         with:
           name: pack-${{ matrix.os }}
           path: out/${{ env.PACK_BIN }}
-  release:
+  build-additional-archs:
     if: ${{ startsWith(github.ref, 'refs/heads/release/') }}
```

Comment:
I figure that it only really makes sense to do if this is a release branch, though happy to make it happen regardless so we can always check the binary and make sure it works